src/libecalc/presentation/yaml/yaml_types/emitters/yaml_venting_emitter.py

```diff
@@ -222,7 +223,9 @@ def get_emissions(
                 .tolist()
             )
             emission_rate = [oil_rate * factor for oil_rate, factor in zip(oil_rates, factors)]
-            emission_rate = Unit.to(self.volume.rate.unit, Unit.TONS_PER_DAY)(emission_rate)
+
+            # Emission factor is kg/Sm3 and oil rate/volume is Sm3/d. Hence, the emission rate is in kg/d:
+            emission_rate = Unit.to(Unit.KILO_PER_DAY, Unit.TONS_PER_DAY)(emission_rate)
 
             emissions[emission.name] = TimeSeriesStreamDayRate(
```

Comment:
I think this function would be simpler if we set up TimeSeries at the start. Then we could use to_unit and implement `__mul__` to handle the units. But, we would have to figure out how to provide the factor with a unit, so it will probably have to wait